### PR TITLE
Ensure edm:object values are HTTP(S) URIs when building thumbnails

### PR DESCRIPTION
**Pull request by anarchivist**
Let me know if you think this needs a test; there are no tests for the `Item` model at all yet. :disappointed_relieved:

### CODE REVIEW

File: app/models/item.rb [hunk 1]
```diff
@@ -147,6 +149,12 @@ def score
 
   private
 
+  def valid_http_uri?(uri)
```
Comment (no-reply) [inline app/models/item.rb]: The lines below are indented in the style used by the Rails team (an extra two spaces for `private`), do we want to do that here for consistency?

**Comment by no-reply** [comment]
:+1:

Tests would be great, but I find it hard to be a stickler about it in this repository. @markbreedlove, @AudreyAltman opinions? Should we get a couple of unit tests in?

**Comment by markbreedlove** [comment]
Maybe just a quick test, since we did the same with the ApplicationController.

**Comment by anarchivist** [comment]
OK, I've added a test to check valid HTTP/HTTPS and invalid URIs. I've also fixed up the whitespace issue that @no-reply raised.

**Comment by markbreedlove** [comment]
OK, fine with me!

**Pull request merged**